linux/runtime.go

```diff
@@ -197,7 +197,8 @@ func (r *Runtime) Create(ctx context.Context, id string, opts plugin.CreateOpts)
 	if err != nil {
 		return nil, err
 	}
-	s, err := newShim(ctx, r.shim, path, namespace, r.remote)
+	sockPath := filepath.Join(string(filepath.Separator), "containerd", "linux", namespace, id)
```

Comment:
true, but given this is in the abstract namespace it won't affect the actual filesystem